tests/path_gateway_ipns_test.go

```diff
@@ -7,6 +7,74 @@ import (
 	. "github.com/ipfs/gateway-conformance/tooling/test"
 )
 
+var (
+	// See fixtures/ipns_records/README.md for information
+	ipnsV1                = "k51qzi5uqu5dm4tm0wt8srkg9h9suud4wuiwjimndrkydqm81cqtlb5ak6p7ku"
+	ipnsV1V2BrokenValueV1 = "k51qzi5uqu5dlmit2tuwdvnx4sbnyqgmvbxftl0eo3f33wwtb9gr7yozae9kpw"
+	ipnsV1V2BrokenSigV2   = "k51qzi5uqu5diamp7qnnvs1p1gzmku3eijkeijs3418j23j077zrkok63xdm8c"
+
+	ipnsV1V2BrokenSigV1     = "k51qzi5uqu5dilgf7gorsh9vcqqq4myo6jd4zmqkuy9pxyxi5fua3uf7axph4y"
+	bodyIPNSV1V2BrokenSigV1 = []byte("v1+v2 with broken signature v1")
+
+	ipnsV1V2     = "k51qzi5uqu5dlkw8pxuw9qmqayfdeh4kfebhmreauqdc6a7c3y7d5i9fi8mk9w"
+	bodyIPNSV1V2 = []byte("v1+v2 record")
+	cidIPNSV1V2  = "bafkqaddwgevxmmraojswg33smq"
+
+	ipnsV2     = "k51qzi5uqu5dit2ku9mutlfgwyz8u730on38kd10m97m36bjt66my99hb6103f"
+	bodyIPNSV2 = []byte("v2-only record")
+	cidIPNSV2  = "bafkqadtwgiww63tmpeqhezldn5zgi"
+)
+
+func TestGatewayIPNSPath(t *testing.T) {
+	tests := SugarTests{
+		{
+			Name: "GET an IPNS Path (V1) from the gateway fails with 5xx",
+			Request: Request().
+				Path("/ipns/{{name}}", ipnsV1),
+			Response: Expect().
+				StatusBetween(500, 599),
+		},
+		{
+			Name: "GET an IPNS Path (V1+V2) with broken ValueV1 from the gateway fails with 5xx",
+			Request: Request().
+				Path("/ipns/{{name}}", ipnsV1V2BrokenValueV1),
+			Response: Expect().
+				StatusBetween(500, 599),
+		},
+		{
+			Name: "GET an IPNS Path (V1+V2) with broken SignatureV1, but valid SignatureV2 succeeds",
+			Request: Request().
+				Path("/ipns/{{name}}", ipnsV1V2BrokenSigV1),
+			Response: Expect().
+				Status(200).
+				Body(bodyIPNSV1V2BrokenSigV1),
+		},
+		{
+			Name: "GET an IPNS Path (V1+V2) from the gateway",
+			Request: Request().
+				Path("/ipns/{{name}}", ipnsV1V2),
+			Response: Expect().
+				Body(bodyIPNSV1V2),
+		},
+		{
+			Name: "GET an IPNS Path (V2) from the gateway",
+			Request: Request().
+				Path("/ipns/{{name}}", ipnsV2),
+			Response: Expect().
+				Body(bodyIPNSV2),
+		},
+		{
+			Name: "GET an IPNS Path (V1+V2) with broken SignatureV2 from the gateway fails with 5xx",
+			Request: Request().
+				Path("/ipns/{{name}}", ipnsV1V2BrokenSigV2),
+			Response: Expect().
+				StatusBetween(500, 599),
```

Comment:
We could implement a `.Status("5xx")` if you need

Reply:
@laurentsenta that is also an option